modules/apps/foundation/portal-search/portal-search-api/src/main/resources/content/Language.properties

```diff
@@ -14,13 +14,16 @@ index-read-only=Index Read Only
 index-status-manager-configuration-name=Index Status Manager
 index-writer-helper-configuration-name=Indexer Writer Helper
 indexer-registry-configuration-name=Search Index Registry
+indexing-intervals=Indexing Intervals
+indexing-intervals-help=Specify the number of documents to be indexed per batch for a given entry class name. The number of documents per batch can be decreased to help prevent Out of Memory errors. Some entry class names may not support this configuration, and will use the default value of 10000.
```

Comment:
@rbohl can you take a quick look at this new lang key? Thanks!

Reply:
@BryanEngler It's pretty good. I tried to shorten it a bit, see what you think: 

indexing-intervals-help=Specify the number of documents to index per batch for a given entry class name. Decreasing the number of documents per batch helps prevent Out of Memory errors. Entry class names that do not support this configuration use the default value of 10000.

Reply:
cool, sounds good, thanks!